client/my-sites/marketplace/components/billing-interval-switcher/index.tsx

```diff
@@ -57,7 +89,25 @@ const BillingIntervalSwitcher: React.FunctionComponent< Props > = ( {
 	return (
 		<Container>
 			<PlansIntervalToggleLabel>{ translate( 'Price' ) }</PlansIntervalToggleLabel>
-			<PlansIntervalToggle intervalType={ billingPeriod } onChange={ onChange } />
+			<PlansIntervalToggle intervalType={ billingPeriod } onChange={ onChange }>
+				{ plugin ? (
+					<PluginAnnualSaving plugin={ plugin }>
+						{ ( annualSaving: { saving: any } ) =>
+							annualSaving.saving ? (
+								<>
+									<PluginAnnualSavingLabelDesktop>
+										(
+										{ sprintf( 'Save %(save)s', {
```

Comment:
It looks like in [the design](https://github.com/Automattic/wp-calypso/issues/59397#issuecomment-1039792675) there's a little space before `(Save...` on desktop and `-` on mobile, can we add that in?

Reply:
<img width="287" alt="Screen Shot 2022-02-21 at 4 50 20 PM" src="https://user-images.githubusercontent.com/1689238/155030344-37e198a2-c874-4ac5-a2dd-23fca11343dd.png">

Reply:
Fixed using Css